api/layer/layer.go

```diff
@@ -380,8 +381,7 @@ func (n *layer) ListBuckets(ctx context.Context) ([]*data.BucketInfo, error) {
 func (n *layer) GetObject(ctx context.Context, p *GetObjectParams) error {
 	var params getParams
 
-	params.oid = p.ObjectInfo.ID
-	params.cid = p.ObjectInfo.CID
+	params.objInfo = p.ObjectInfo
```

Comment:
I suppose we can keep the old `getParams` fields. And extend `GetObjectParams` by `bktInfo` and pass it (`bktInfo`) as the second parameter to `n.initObjectPayloadReader` function

Reply:
Removed this with dragging of `bktInfo` through params. Added a separated commit to make it easier to view changes